programs/programs/urgent_needs/il/il_beacon.py

```diff
@@ -0,0 +1,22 @@
+from ..base import UrgentNeedFunction
+
+
+class il_beacon(UrgentNeedFunction):
+    dependencies = ["age"]
+    min_age = 5
+    max_age = 21
+
+    def eligible(self):
+        # Check if household has marked mental health or child development help
```

Comment:
I think we can remove these need conditions from this urgent need function, and instead configure this in django admin.
<img width="1762" height="536" alt="image" src="https://github.com/user-attachments/assets/a01fc97b-650f-4b6a-9380-a1cba37be001" />